### PR TITLE
Tentative changes from final version used for photoacclimation experiments

### PR DESCRIPTION
**Pull request by andrew-c-ross**
This PR would make COBALT equivalent to the version found in Charlie's generic_COBALT_08222023_control.F90. Some of the changes in this PR are specific to the needs of the global photoacclimation runs, however, and shouldn't be merged. Reviews would be appreciated to identify what should or shouldn't be merged.

### CODE REVIEW

File: generic_tracers/generic_COBALT.F90 [hunk 1]
```diff
@@ -6341,14 +6341,14 @@ subroutine user_add_params
...
-    call g_tracer_add_param('p_2_n_min_Lg', phyto(LARGE)%p_2_n_min,1.0/31.0 )               ! mol P mol N-1
-    call g_tracer_add_param('p_2_n_slope_Lg', phyto(LARGE)%p_2_n_slope, 0.048*1.0e6)        ! mol P mol N-1 mol P-1 kg
+    call g_tracer_add_param('p_2_n_min_Sm', phyto(SMALL)%p_2_n_min,1.0/23.0 )               ! mol P mol N-1
+    call g_tracer_add_param('p_2_n_slope_Sm', phyto(SMALL)%p_2_n_slope, 0.0*1.0e6)        ! mol P mol N-1 mol P-1 kg 
+    call g_tracer_add_param('p_2_n_max_Sm', phyto(SMALL)%p_2_n_max,1.0/23.0 )               ! mol P mol N-1
+    call g_tracer_add_param('p_2_n_min_Md', phyto(MEDIUM)%p_2_n_min,1.0/18.0 )              ! mol P mol N-1
+    call g_tracer_add_param('p_2_n_slope_Md', phyto(MEDIUM)%p_2_n_slope, 0.0*1.0e6)       ! mol P mol N-1 mol P-1 kg
+    call g_tracer_add_param('p_2_n_max_Md', phyto(MEDIUM)%p_2_n_max,1.0/18.0 )              ! mol P mol N-1
+    call g_tracer_add_param('p_2_n_min_Lg', phyto(LARGE)%p_2_n_min,1.0/14.0 )               ! mol P mol N-1
+    call g_tracer_add_param('p_2_n_slope_Lg', phyto(LARGE)%p_2_n_slope, 0.0*1.0e6)        ! mol P mol N-1 mol P-1 kg
```
Comment (andrew-c-ross) [inline generic_tracers/generic_COBALT.F90:6351]: I think I see what's going on here now... this is effectively disabling the variable n:p, right? So this should not be merged.

**Comment by yichengt900** [comment]
Thanks @andrew-c-ross! @charliestock, based on @nikizadehgfdl's comments during the `COBALTv3 Focused Doc and Dev` meeting, it seems that we can overwrite parameters using field_table without recompiling the code. This raises the following question: do we still want to update the default values to match those used in the manuscript, purely for the sake of consistency and reference?

**Comment by yichengt900** [comment]
@charliestock, following our discussion, I think we will eventually go down the route of providing users with our recommended BGC parameters through a separate file, rather than editing the default values in the code.  If so, I will close this PR for now and keep it as a reference. @andrew-c-ross are you ok with that?

**Comment by andrew-c-ross** [comment]
I think that sounds ok. Yeah I will keep my branch around, so with that branch and this closed PR we can always compare values if needed.

**Pull request closed without merging**